apps/worker/src/app/workflow/services/metric-queue-active.service.ts

```diff
@@ -89,51 +73,37 @@ export class MetricQueueService extends QueueService<Record<string, never>> {
     return async () => {
       return await new Promise<void>(async (resolve, reject): Promise<void> => {
         Logger.verbose('metric job started', LOG_CONTEXT);
+        const deploymentName = process.env.FLEET_NAME ?? 'default';
 
         try {
           for (const queueService of this.token_list) {
-            const metrics = await queueService.bullMqService.getQueueMetrics();
-
-            const completeNumber = metrics.completed.count;
-            const failNumber = metrics.failed.count;
-
-            const successMetric: IMetric = {
-              count: metrics.completed.count,
-              total: completeNumber == 0 ? 0 : sum(metrics.completed.data),
-              min: completeNumber == 0 ? 0 : min(metrics.completed.data),
-              max: completeNumber == 0 ? 0 : max(metrics.completed.data),
-              sumOfSquares: completeNumber == 0 ? 0 : sumOfSquares(metrics.completed.data),
-            };
-
-            const failMetric: IMetric = {
-              count: metrics.failed.count,
-              total: failNumber == 0 ? 0 : sum(metrics.failed.data),
-              min: failNumber == 0 ? 0 : min(metrics.failed.data),
-              max: failNumber == 0 ? 0 : max(metrics.failed.data),
-              sumOfSquares: failNumber == 0 ? 0 : sumOfSquares(metrics.failed.data),
-            };
-
             const waitCount = await queueService.bullMqService.queue.getWaitingCount();
             const delayedCount = await queueService.bullMqService.queue.getDelayedCount();
             const activeCount = await queueService.bullMqService.queue.getActiveCount();
 
-            if (process.env.NOVU_MANAGED_SERVICE === 'true') {
-              nr.recordMetric(`MetricQueueService/${queueService.name}/completed`, successMetric);
-              nr.recordMetric(`MetricQueueService/${queueService.name}/failed`, failMetric);
-              nr.recordMetric(`MetricQueueService/${queueService.name}/waiting`, waitCount);
-              nr.recordMetric(`MetricQueueService/${queueService.name}/delayed`, delayedCount);
-              nr.recordMetric(`MetricQueueService/${queueService.name}/active`, activeCount);
+            if (process.env.NOVU_MANAGED_SERVICE === 'true' && process.env.NEW_RELIC_LICENSE_KEY.length !== 0) {
+              Logger.verbose('active length', process.env.NEW_RELIC_LICENSE_KEY.length);
```

Comment:
This might throw an error when undefined
```suggestion
            if (process.env.NOVU_MANAGED_SERVICE === 'true' && process.env.NEW_RELIC_LICENSE_KEY) {
              Logger.verbose('active length', process.env.NEW_RELIC_LICENSE_KEY.length);
```

Suggest doing it like this